### ISSUE TITLE
Does this compile on Linux?

### ISSUE THREAD

**Issue by djnotes** [zesterer/vast#1]
Does this code compile on Linux? It looks it only works on Windows right?

**Comment by zesterer** [comment]
Yes, I built it for Linux. This is an old version of the project, however. I'll push the new one within a week.

**Comment by djnotes** [comment]
Thanks. That'll be great!

**Comment by zesterer** [comment]
New repository for the new version here: https://github.com/zesterer/vast-dev
